src/api/api.chart.js

```diff
@@ -74,6 +75,7 @@ extend(Chart.prototype, {
 			$$.charts.splice($$.charts.indexOf(this), 1);
 
 			// clear timers
+			$$.svg.selectAll(`.${CLASS.target}`).transition();
```

Comment:
It should be calling `.interrupt()` rather than .transition(). Also the targets should be pointing all the descendants of `<svg>`.

```js
$$.svg.select("*").interrupt();
```
Take a look on d3's doc
- https://github.com/d3/d3-transition#selection_interrupt